security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -36,6 +36,8 @@ The following advisory is available for the OpenShift Compliance Operator 1.5.0:
 
 * Previously, for ROSA HCP systems, profile listings were incorrect. This update allows the Compliance Operator to provide correct profile output. (link:https://issues.redhat.com/browse/OCPBUGS-34535[*OCPBUGS-34535*])
 
+* With this release, namespaces which do not need `networkpolicy` defined are whitelisted. (link:https://issues.redhat.com/browse/cmp-2543[*CMP-2543*])
```

Comment:
a note about conscious language to avoid blacklist/whitelist and similar terms: https://redhat-documentation.github.io/supplementary-style-guide/#_blacklist_and_whitelist

suggestion, but it's just one variant so pick a flavor that works best 😄:
```suggestion
* With this release, namespaces which do not need `networkpolicy` defined are allowed. (link:https://issues.redhat.com/browse/cmp-2543[*CMP-2543*])
```

Reply:
Maybe update it to 
`With this release, it is allowed use a variable ocp4-var-network-policies-namespaces-exempt-regex to exclude the namespace which do not need `networkpolicy``

Reply:
@xiaojiey @jldohmann @Vincent056  I'd like to go with Vincent's suggestion, if that is OK. It seems to be the most specific of all.

Reply:
sounds good to me